docs/reference/synonyms/apis/synonyms-apis.asciidoc

```diff
@@ -29,8 +34,25 @@ Use the following APIs to manage synonyms sets:
 * <<list-synonyms-sets>>
 * <<delete-synonyms-set>>
 
+[discrete]
+[[synonym-rules-apis]]
+=== Synonym rules APIs
+
+Synonyms sets are composed of synonym rules.
+Each synonym rule defines a group of words that act as synonyms.
+You can manage synonym rules for a specific synonyms set instead of updating the complete synonyms set.
```

Comment:
```suggestion
You can manage synonym rules for a specific synonym set instead of updating the complete set.
```

Reply:
How about this modification:
You can manage synonym rules => You can update individual synonym rules